iroh-relay/src/protos/relay.rs

```diff
@@ -51,104 +36,7 @@ pub(crate) const PING_INTERVAL: Duration = Duration::from_secs(15);
-
-/// The one byte frame type at the beginning of the frame
-/// header. The second field is a big-endian u32 describing the
-/// length of the remaining frame (not including the initial 5 bytes)
-#[derive(Debug, PartialEq, Eq, num_enum::IntoPrimitive, num_enum::FromPrimitive, Clone, Copy)]
-#[repr(u8)]
-pub enum FrameType {
-    /// magic + 32b pub key + 24B nonce + bytes
-    ClientInfo = 2,
-    /// 32B dest pub key + packet bytes
-    SendPacket = 4,
-    /// v0/1 packet bytes, v2: 32B src pub key + packet bytes
-    RecvPacket = 5,
-    /// no payload, no-op (to be replaced with ping/pong)
-    KeepAlive = 6,
-    /// 1 byte payload: 0x01 or 0x00 for whether this is client's home node
-    NotePreferred = 7,
-    /// Sent from server to client to signal that a previous sender is no longer connected.
-    ///
-    /// That is, if A sent to B, and then if A disconnects, the server sends `FrameType::PeerGone`
-    /// to B so B can forget that a reverse path exists on that connection to get back to A
-    ///
-    /// 32B pub key of peer that's gone
-    PeerGone = 8,
-    /// Frames 9-11 concern meshing, which we have eliminated from our version of the protocol.
-    /// Messages with these frames will be ignored.
-    /// 8 byte ping payload, to be echoed back in FrameType::Pong
-    Ping = 12,
-    /// 8 byte payload, the contents of ping being replied to
-    Pong = 13,
-    /// Sent from server to client to tell the client if their connection is
-    /// unhealthy somehow.
-    ///
-    /// Currently this is used to indicate that the connection was closed because of authentication issues.
-    Health = 14,
-
-    /// Sent from server to client for the server to declare that it's restarting.
-    /// Payload is two big endian u32 durations in milliseconds: when to reconnect,
-    /// and how long to try total.
-    ///
-    /// Handled on the `[relay::Client]`, but currently never sent on the `[relay::Server]`
-    Restarting = 15,
-    /// Unknown frame type
-    #[num_enum(default)]
-    Unknown = 255,
-}
-
-impl std::fmt::Display for FrameType {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
-        write!(f, "{self:?}")
-    }
-}
-
-#[derive(Debug, Serialize, Deserialize, MaxSize, PartialEq, Eq)]
-pub(crate) struct ClientInfo {
-    /// The relay protocol version that the client was built with.
-    pub(crate) version: usize,
-}
-
-/// Protocol send errors.
-#[common_fields({
-    backtrace: Option<Backtrace>,
-    #[snafu(implicit)]
-    span_trace: n0_snafu::SpanTrace,
-})]
-#[allow(missing_docs)]
-#[derive(Debug, Snafu)]
-#[non_exhaustive]
-pub enum SendError {
-    #[snafu(transparent)]
-    Io { source: std::io::Error },
-    #[snafu(transparent)]
-    Timeout { source: time::Elapsed },
-    #[snafu(transparent)]
-    ConnSend { source: ConnSendError },
-    #[snafu(transparent)]
-    SerDe { source: postcard::Error },
-}
+pub(crate) const PER_CLIENT_SEND_QUEUE_DEPTH: usize = 512;
```

Comment:
This comment isn't actionable.  But :exploding_head:
Used to be 32 in the tailscale code by the looks of the old comment.  I suspect this was turned up at some point because it make a backpressure problem go away for a particular example.  But we've actually improved backpressure since.  Could we create an issue to investigate this and see if this number can be turned down?